test/e2e/resourceinterpreter_test.go

```diff
@@ -226,4 +227,46 @@ var _ = ginkgo.Describe("Resource interpreter webhook testing", func() {
 			}
 		})
 	})
+
+	ginkgo.Context("InterpreterOperation InterpretHealth testing", func() {
+		ginkgo.It("InterpretHealth testing", func() {
+			resourceBindingName := names.GenerateBindingName(workload.Kind, workload.Name)
+
+			SetReadyReplicas := func(readyReplicas int32) {
+				for _, cluster := range framework.ClusterNames() {
+					clusterDynamicClient := framework.GetClusterDynamicClient(cluster)
+					gomega.Expect(clusterDynamicClient).ShouldNot(gomega.BeNil())
+					memberWorkload := framework.GetWorkload(clusterDynamicClient, workloadNamespace, workloadName)
+					memberWorkload.Status.ReadyReplicas = readyReplicas
+					framework.UpdateWorkload(clusterDynamicClient, memberWorkload, cluster, "status")
+				}
+			}
+
+			CheckResult := func(result workv1alpha2.ResourceHealth) interface{} {
+				return func(g gomega.Gomega) (bool, error) {
+					rb, err := karmadaClient.WorkV1alpha2().ResourceBindings(workload.Namespace).Get(context.TODO(), resourceBindingName, metav1.GetOptions{})
+					g.Expect(err).NotTo(gomega.HaveOccurred())
+					g.Expect(len(rb.Status.AggregatedStatus)).Should(gomega.Equal(3))
```

Comment:
If this assertion fails, the loop wait will exit, which should not be what we want.
Besides, we can use `len(framework.ClusterNames())` to replace number 3.